lib/Game.rb

```diff
@@ -0,0 +1,65 @@
+
+class Game
+    include UserInterface
+    def initialize(board, player1, player2)
+        @board =  board
+        @player_1 = player1
+        @player_2 = player2
+        @current_player = @player_1
+    end
+    WIN_COMBO = [
+      [0, 1, 2],
+      [3, 4, 5],
+      [6, 7, 8],
+      [0, 3, 6],
+      [1, 4, 7],
+      [2, 5, 8],
+      [0, 4, 8],
+      [6, 4, 2]
+     ]
+
+    def play
+        loop do 
+            self.show_board(@board)
+            @current_player.ask_move 
+            break if is_game_over?
+            @current_player = @current_player == @player_1 ? @player_2 : @player_1
+        end
+    end
+
+    def is_game_over?
+        is_victory? || is_draw?
+    end
+
+    private
+        def is_victory?
+            if winner? 
+                self.show_board(@board)
+                self.print_winner(@current_player)
+                true
+            else
```

Comment:
No need to put an `else` statement here because should the `if` statement above doesn't hold true then it will, by default become false. So you can just do this:

```
if winner?
  show_board(board)
  return true
end
false
```